www/front_src/src/App/index.tsx

```diff
@@ -34,13 +35,16 @@ const useStyles = makeStyles((theme) => ({
     zIndex: 1500,
   },
   mainContent: {
-    backgroundColor: theme.palette.background.paper,
-    height: `calc(100vh - ${theme.spacing(headerHeight + headerPadding)})`,
+    height: `calc(100vh - ${theme.spacing(headerHeight)})`,
     width: '100%',
   },
   wrapper: {
     alignItems: 'stretch',
+    backgroundColor: equals(theme.palette.mode, ThemeMode.dark)
+      ? '#2c2c2c'
```

Comment:
This must use a color from the theme

Reply:
I replaced hexa code with theme.palette.background.paper (as light theme).